tests/ipfwd/test_nhop_group.py

```diff
@@ -270,8 +318,6 @@ def test_nhop(request, duthost, tbinfo):
     - clean up
     - Verify no errors and crash
     """
-    skip_release(duthost, ["201811", "201911"])
```

Comment:
Don't we need this?

Reply:
this PR will go only in master of sonic-mgmt.  Will add this back once if needed if we decide to back port to other branch.